txnkv/transaction/txn.go

```diff
@@ -596,6 +596,19 @@ func (txn *KVTxn) LockKeys(ctx context.Context, lockCtx *tikv.LockCtx, keysInput
 		// it before initiating an RPC request.
 		ctx = interceptor.WithRPCInterceptor(ctx, txn.interceptor)
 	}
+
+	if lockCtx.LockOnlyIfExists {
+		if !lockCtx.ReturnValues {
+			return errors.New("LockOnlyIfExists is set for Lock Context, but ReturnValues is not set")
```

Comment:
Better to wrap the error information with the transaction and key information and put it in `error/errors.go`.

Reply:
@cfzjywxk PTAL, thanks